src/amo/components/GuidesAddonCard/index.js

```diff
@@ -153,20 +149,12 @@ export class GuidesAddonCardBase extends React.Component<InternalProps> {
                 puffy={false}
               />
             )}
-            {/* TODO: See https://github.com/mozilla/addons-frontend/issues/6901. */}
-            {showGetFirefoxButton && (
-              <Button
-                buttonType="confirm"
-                href={`https://www.mozilla.org/firefox/new/${makeQueryStringWithUTM(
-                  {
-                    utm_content: addon.guid,
-                  },
-                )}`}
-                puffy
-                className="Button--get-firefox"
-              >
-                {i18n.gettext('Only with Firefox—Get Firefox Now')}
-              </Button>
+            {addon && (
```

Comment:
Is this still needed given that L105 there is already this condition?

Reply:
I believe you can remove it.

Reply:
good point